src/ParentAwareNodeInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Dhii\Data\Tree;
+
+use Dhii\Data\Hierarchy\ParentAwareInterface;
+
+/**
+ * A node that can have its parent retrieved.
+ *
+ * @since [*next-version*]
+ */
+interface ParentAwareNodeInterface extends
+        NodeInterface,
+        ParentAwareInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     *
+     * @return BasicNodeInterface|null The node's parent, if any.
```

Comment:
`BasicNodeInterface` is no longer in this package. I recommend changing this to `NodeInterface`.